docs/AA--Javascript-in-EE/value-objects/server-date-time.md

```diff
@@ -0,0 +1,54 @@
+## ServerDateTime Value Object
+
+The `ServerDateTime` value object represents an instance in time.  It is identical in behaviour to the [`DateTime`](datetime.md) value object (it extends it) except that it automatically takes care of the timezone or offset currently set on the server.
+
+This is needed because if a WordPress site has a UTC offset set instead of a timezone string, the timezone string will be empty.  So, using `DateTime` only, client code would always have to check whether timezone string is empty to derive how to instantiate the value object (with an offset, or with a timezone string).  `ServerDateTime` takes care of this automatically so client code just has to provide the value for the constructor method being used and (optionally) the locale (which defaults to the server locale if not provided).
+
+For more details on all the methods and properties available, refer to the [`DateTime` documentation](datetime.md).  Otherwise, there's only three methods overridden by this value object.
+
+### CONSTRUCTOR
+
+The constructor simply receives the following two arguments:
+
+- `iso8601DateString`: This should be an `ISO8601` formatted date/time string.  If offset is included the string, this will be converted to whatever the timezone/offset is on the server.
```

Comment:
how about:

> This should be an ISO 8601 formatted date/time string.

ie: `ISO8601` should have a space and does not need to be represented as code since it is valid text